### PR TITLE
Added GetToken to DolbyIOManager

### PR DESCRIPTION
**Pull request by 4thlabs**

### CODE REVIEW

File: scripts/DolbyIOManager.cs [hunk 1]
```diff
@@ -24,6 +30,42 @@ public class DolbyIOManager : MonoBehaviour
...
+        /// <param name="key">The customer key.</param>
+        /// <param name="secret">The customer secret.</param>
+        /// <returns>An asynchronous task containing the token.</returns>
+        public static async Task<string> GetToken(string key, string secret)
+        {
+            string result = "";
+
+            using (var client = new HttpClient())
+            {
+                var request = new HttpRequestMessage(HttpMethod.Post, "https://api.dolby.io/v1/auth/token");
```
Comment (jfan-dolby) [inline scripts/DolbyIOManager.cs:47]: We are still on https://session.voxeet.com/v1/oauth2/token for the client access token, this one is for REST API (api token).

Comment (4thlabs) [reply scripts/DolbyIOManager.cs:47]: Ho! No problem

File: scripts/DolbyIOManager.cs [hunk 2]
```diff
@@ -24,6 +30,42 @@ public class DolbyIOManager : MonoBehaviour
         [Tooltip("Indicates if the DolbyIOManager should automatically close the session on application quit.")]
         public bool AutoCloseSession = true;
 
+        /// <summary>
+        /// For convenience during early development and prototyping, a GetToken (linked) method is provided for you to 
+        /// acquire a client access token directly from the application. However, please note Dolby does not recommend 
+        /// using this mechanism in the production software for (security best practices)[https://docs.dolby.io/communications-apis/docs/guides-client-authentication] reasons.
+        /// </summary>
+        /// <param name="key">The customer key.</param>
```
Comment (jfan-dolby) [inline scripts/DolbyIOManager.cs:38]: Let's go with "The App Key" and "The App Secret" to be consistent with what we show in the dashboard.

Comment (4thlabs) [reply scripts/DolbyIOManager.cs]: done

**Review by jfan-dolby — APPROVED** [review]

**Pull request merged**